css/types/image.json

```diff
@@ -290,6 +290,39 @@
                 "deprecated": false
               }
             },
+            "color_interpolation_method": {
+              "__compat": {
+                "description": "Interpolation Method / Color Spaces",
```

Comment:
"Cylindrical Color Space Hue Interpolation"?
"Hue Interpolation Method"?

Reply:
TL;DR; 
My vote is for `color space interpolation`

VR;NR; (Very long, not reading):
I don’t think BCD is differentiating between the different color spaces. BCD is just listing whether all supported color spaces support color interpolation or not. 

We can remove the word “method” to make it shorter.

Within a property or feature value, such as in gradients,  “color space interpolation” might be best. My original thought was “color interpolation” would be enough, but it isn't, as readers will assume that it just means "it goes from red to blue whichever way the browser decides, whereas adding the "space" lets the reader know there is something else there. 

In terms of the color page itself, we may want to list which color space interpolations are supported, so "sRGB color space interpolation" would be appropriate on that page only, as that is the only space where we may list the different various spaces individually.

Reply:
There are two feature encapsulated here :
- in which color space are you interpolating?
- how are you interpolating?

All browsers shipped or are shipping these simultaneously and authors are also learning about these simultaneously. Both are directly related to color interpolation. They are also defined in the same spec module.
But both do very different things.

- The color space setting determines the underlying color model that will be used when interpolating.
- The hue interpolation method determines that path that will be taken between two colors.

`color space interpolation` places emphasis on the color spaces and doesn't mention/imply the hue interpolation method.

---------

The more I think about it, the more I am in favor of splitting these up in two features.
That makes it easier to have a descriptive name that isn't overly long and verbose.

It will also help authors to talk about these and look them up :

> Q : I want to display a color wheel
> A : Have you tried the setting the `hue interpolation method` to `longer`?

> Q : I don't like the mid point of my gradient
> A : Have you tried with a different `interpolation color space`?

Reply:
Splitting this up sounds good to me

Reply:
I've split these up and updated the comment at the top to reflect this.